submissions/pozzitive11/frogger-game/app.js

```diff
@@ -0,0 +1,117 @@
+const tileSize = {
+  width: 101,
+  height: 85,
+};
+
+const enemySpeed = {
+  first: 90,
+  second: 230,
+  third: 150,
+};
+
+const boardSize = {
+  top: 0,
+  right: tileSize.width * 5,
+  bottom: tileSize.height * 6,
+  left: 0,
+};
+
+class gameObjectRender {
+  render() {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+  }
+}
+class Enemy extends gameObjectRender {
+  constructor(y, speed, sprite) {
+    super(y, sprite);
```

Comment:
Super class doesn't have a constructor. This operation is effectively void. Same is applicable to the `Player` class. However the idea to have similar properties and similar behaviour in a base class is great.
Can you fix this and implement shared properties and methods via the base class?